docs/tools/fbsql/fbsql-home.md

```diff
@@ -42,6 +43,7 @@ The following flags can be provided when running fbsql. None of the flags are re
 | `--org-id` | Specified the Organization ID to use. Organizations are a concept used in FeatureBase Cloud, and in that case they are determined automatically based on user authorization. They are exposed here in case on-prem installations want to mimic that functionality. | |
 | `--password` | Password for FeatureBase Cloud access. | |
 | `-p`<br>`--port` | Specifies the TCP port or the local Unix-domain socket file extension on which FeatureBase is listening for connections. | Attempts to detect `10101` or `8080` (for serverless) | 
+| `-P`<br>`--pset` | Specifies printing options, in the style of `\pset`. Note that here you have to separate name and value with an equal sign instead of a space. For example, to set the output format to CSV, you could write -P format=csv. | |
```

Comment:
@travisturner is this capital P vs lowercase p for port? Not sure if we want the short flag, but I leave it up to you!

Reply:
That's correct. I'm just trying to follow `psql` as much as I can, and that's the flags it uses.